vulnfeeds/cves/versions.go

```diff
@@ -62,6 +64,74 @@ func extractGitHubCommit(link string) *FixCommit {
 	}
 }
 
+func hasVersion(validVersions []string, version string) bool {
+	return versionIndex(validVersions, version) != -1
+}
+
+func versionIndex(validVersions []string, version string) int {
+	for i, cur := range validVersions {
+		if cur == version {
+			return i
+		}
+	}
+	return -1
+}
+
+func nextVersion(validVersions []string, version string) string {
+	idx := versionIndex(validVersions, version) + 1
+	if idx < len(validVersions) {
+		return validVersions[idx]
+	}
+
+	log.Printf("Warning: %s is not a valid version", version)
```

Comment:
this is a confusing error message, since versions that are not in `validVersions` have a `nextVersion` of `validVersions[0]`

Should this be something like "Warning: next version after %s is not a valid version"

Reply:
Woops, that's a mistake. If `version` does not exist, we should return `""` instead. I fixed this and made error message clearer.

Good catch, thanks!